uiowa/uiowa
# Changed Node image style to Medium - No crop and Photoswipe image sty…

Resolves https://github.com/uiowa/uiowa/issues/4827

How to test

1. blt ds --site=sandbox.uiowa.edu
2. https://sandbox.prod.drupal.uiowa.edu/single-image-gallery
3. Check to make sure that the node image styles is "medium - no crop" and the Photoswipe image is the original image.
4. Available to test on dev: https://sandbox.dev.drupal.uiowa.edu/single-image-gallery
